web/js/codeworld.js

```diff
@@ -675,6 +675,15 @@ function help() {
         allowEscapeKey: true,
         allowOutsideClick: true,
         showConfirmButton: false,
+    }, function(confirmed){
```

Comment:
Please run the format.sh script that I added in the root codeworld directory.  It reformats to a consistent style everywhere, and updates some coding style things... for example, using arrow functions instead of lambdas, and let instead of var.

Reply:
Done